t/plugin/response-rewrite.t

```diff
@@ -699,3 +699,512 @@ X-A: 127.0.0.1
 X-B: from 127.0.0.1 to 127.0.0.1:1980
 --- no_error_log
 [error]
+
+
+
+=== TEST 25:  add plugin with valid filters
```

Comment:
Let's move the new test to t/plugin/response-rewrite2.t. Otherwise this test file is too long.